### DISCUSSION TITLE
Kimai break time check

### DISCUSSION THREAD

**Discussion by kathy2309** [kimai/kimai#3807] [Questions & Answers]
We use Kimai exclusively for time tracking of our employees. Via the break time check I can see which employee has too short break times. I would also like to see how much break was taken total on every day and how much time is still missing. 
Is it possible to deduct the missing break time from the working time?

**Comment by kevinpapst** [comment]
Sounds as if you are talking about the Controlling plugin? Contact the developer directly. This repo is about Kimai core functionality.
